fast_forward/universe_handler.py

```diff
@@ -12,9 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from collections import defaultdict
+import numpy as np
 import MDAnalysis as mda
 from MDAnalysis import transformations
 
+
+def dim2lattice(x, y, z, alpha=90, beta=90, gamma=90):
```

Comment:
x,y,z are the xyz components of positions I assume so their shape must be a one dimensional array? Is it worth to numba this bit of code or is it fast?

Reply:
this is only done on the box for every frame and it is needed in the initialization. you'll win a few microseconds in total, maybe. You could see if calculating the reciprocal coordinates with numba shaves off a second or so, but I guess it won't be significant on trajectories that fit into memory anyway.

Reply:
I suspected as much but was still curious. Let's not over-engineer this one then